menas/src/main/scala/za/co/absa/enceladus/rest/services/DatasetService.scala

```diff
@@ -20,7 +20,9 @@ import org.springframework.stereotype.Service
 import za.co.absa.enceladus.model.{Dataset, UsedIn}
 import za.co.absa.enceladus.rest.repositories.DatasetMongoRepository
 
-import scala.concurrent.Future
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Future}
```

Comment:
Why Await and Duration? We want stuff to be async

Reply:
Resolved